.gitlab-ci.yml

```diff
@@ -6,11 +6,7 @@
 
 
 stages:
-  - check
```

Comment:
Why does this PR change the CI?

Reply:
I will revert it back after merging with master